packages/env-utils/package.json

```diff
@@ -28,16 +28,18 @@
         "prepublish": "yarn tsx ../../scripts/prepublish.js"
     },
     "dependencies": {
-        "@suite-common/wallet-constants": "workspace:*",
-        "expo-constants": "15.4.5",
         "ua-parser-js": "^1.0.37"
     },
     "peerDependencies": {
+        "expo-constants": "*",
         "expo-localization": "*",
         "react-native": "*",
         "tslib": "^2.6.2"
     },
     "peerDependenciesMeta": {
+        "expo-constants": {
+            "optional": true
+        },
```

Comment:
does this work for all the packages? npm, yarn, pnpm? just curious because this is the first time I see it.

Reply:
I think it has pretty good support and it's already used here for `expo-localization` and `react-native`